app/src/pages/inside/testCaseLibraryPage/testCaseList/testCaseNameCell/testCaseNameCell.tsx

```diff
@@ -26,18 +26,20 @@ interface TestCaseNameCellProps {
   priority: TestCasePriority;
   name: string;
   tags: string[];
+  onRowClick: () => void;
 }
 
-export const TestCaseNameCell = ({ priority, name, tags }: TestCaseNameCellProps) => {
+export const TestCaseNameCell = ({ priority, name, tags, onRowClick }: TestCaseNameCellProps) => {
   return (
-    <div className={cx('name-section')}>
+    <div className={cx('name-section')} onClick={onRowClick}>
       <PriorityIcon priority={priority} className={cx('name-icon')} />
+
```

Comment:
We discussed earlier to not introduce empty lines in markup

Reply:
removed